README.md

```diff
@@ -17,6 +17,15 @@ Jib is a Maven plugin for building container images for your Java applications.
 
 * **Native** - Reduce your CLI dependencies. Build your Docker image from within Maven <!--or Gradle--> and push to any registry of your choice. No more writing Dockerfiles and calling docker build/push.
 
+## Upcoming Features
+
+These features will be added in later releases.
```

Comment:
I know what you're trying to do here, but the wording feels a little strange to me.
What if you were more explicit that these features are not supported?
Like changing this line to:
These features are currently not supported but will be coming in later releases.

Reply:
Okay added the `currently not supported` phrase!